### DISCUSSION TITLE
Export using sync driver?

### DISCUSSION THREAD

**Discussion by phh** [filamentphp/filament#11033] [Help]
Hi there!

We're looking into migrating from the pxlrbt/filament-excel package to using the built in export action instead.

Are there any way to use like the sync driver to get instant export instead of it being a background job?

**Comment by Fanom2813** [comment]
is there an option for this ?

**Comment by Youssef-Amjad** [comment]
> Hi there!
> 
> We're looking into migrating from the pxlrbt/filament-excel package to using the built in export action instead.
> 
> Are there any way to use like the sync driver to get instant export instead of it being a background job?

Is there anything new in this proposal? I desperately need it.
